UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs

```diff
@@ -74,7 +74,8 @@ private void OnWearing(RegisterPlayer player)
 
 		private void OnReleasing()
 		{
-			if (!isAlive || isToy)
+			// check if gameObject is active because gameObject needs to be active to StartCoroutine
+			if (!isAlive || isToy || !gameObject.activeInHierarchy)
 			{
```

Comment:
Coroutine couldn't be started because the the game object 'FacehuggerMask' is inactive!
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
Clothing.FacehuggerImpregnation:OnReleasing() (at /github/workspace/UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs:82)
Clothing.FacehuggerImpregnation:OnInventoryMoveServer(InventoryMove) (at /github/workspace/UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs:139)
Inventory:ServerPerformRemove(InventoryMove, Pickupable) (at /github/workspace/UnityProject/Assets/Scripts/Inventory/Inventory.cs:414)
Inventory:ServerPerform(InventoryMove) (at /github/workspace/UnityProject/Assets/Scripts/Inventory/Inventory.cs:196)
Inventory:ServerDespawn(ItemSlot) (at /github/workspace/UnityProject/Assets/Scripts/Inventory/Inventory.cs:55)
ItemSlot:Free(ItemStorage) (at /github/workspace/UnityProject/Assets/Scripts/Inventory/ItemSlot.cs:452)
ItemStorage:OnDestroy() (at /github/workspace/UnityProject/Assets/Scripts/Inventory/ItemStorage.cs:91)